File: PooCC02CAbel/Semana03/RelojJava/DisplayReloj.java
import java.util.concurrent.TimeUnit;

public class DisplayReloj {
    public static void main(String[] args) {
        DisplayNumero hora = new DisplayNumero(17, 23);
        DisplayNumero minutos = new DisplayNumero(53,59);
        DisplayNumero segundos = new DisplayNumero(3, 59);
        while (true) {
            if (segundos.valor == 0) {
                minutos.incrementar();
            }
            if  ((minutos.valor == 0 ) && (segundos.valor == 0)){
                hora.incrementar();
            }
            try {
                new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor();
            } catch (Exception e) {}
            System.out.println(hora.valor + " : " + minutos.valor + " : " + segundos.valor);
            try {
                TimeUnit.SECONDS.sleep(1);
            } catch(Exception e) {}
            segundos.incrementar();
        }
    }
}

File: PooCC02CAbel/Semana03/RelojPython/DisplayReloj.py
from DisplayNumero import *
import time
from os import system
class DisplayReloj:
    def main():
        hora = DisplayNumero(11, 23)
        minutos = DisplayNumero(3, 59)
        segundos = DisplayNumero(1, 59)
        while(True):
            if segundos.valor == 0:
                minutos.incrementar()
            if minutos.valor == 0 and segundos.valor == 0:
                hora.incrementar()
            system("cls")
            print(f"{hora.valor} : {minutos.valor} : {segundos.valor}")
            time.sleep(1)
            segundos.incrementar()
    
    main()